packages/language/src/v1beta1/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var languageServiceApi = require('./language_service_api');
+
+function v1beta1(options) {
```

Comment:
I modified the handling of the options a bit, and I believe this file is necessary.

In case that an API consists of multiple services (like pubsub -- it has Publisher and Subscriber), I think it's more natural to have this index.js, which creates the auth client (through GAX library) and passes to the service client files.

Please take another look.